packages/react-integration/demo-app-ts/src/components/demos/TableDemo/TableSelectableDemo.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { Table, TableHeader, TableBody, TableProps, headerCol, ICell, IRow } from '@patternfly/react-table';
 import { Checkbox } from '@patternfly/react-core';
+import '@patternfly/patternfly/utilities/Spacing/spacing.css';
```

Comment:
What classes are we using that we need this import for?

Reply:
[`pf-u-mb-lg`](https://github.com/patternfly/patternfly-react/pull/3296/files#diff-1719150855fcc83a974c78f98e5b276cR76)